index.bs

```diff
@@ -306,24 +318,26 @@ disabled for a given browsing context.
 <div algorithm>
 
 To determine if an <dfn export>event is enabled</dfn> given |session|,
-|event name| and |browsing context|:
+|event name| and |browsing contexts|:
```

Comment:
Given event name and browsing contexts, does the event counts as enabled only if it's enabled for all given browsing contexts?

Reply:
No; it's for any.